config/crd/bases/http.keda.sh_httpscaledobjects.yaml

```diff
@@ -162,6 +162,22 @@ spec:
                   metric value
                 format: int32
                 type: integer
+              timeouts:
+                description: (optional) Timeouts that override the global ones
+                properties:
+                  conditionWait:
+                    description: How long to wait for the backing workload to have
+                      1 or more replicas before connecting and sending the HTTP request
+                      (Default is set by the KEDA_CONDITION_WAIT_TIMEOUT environment
+                      variable)
+                    type: string
+                  responseHeader:
+                    description: How long to wait for the backing workload to have
+                      1 or more replicas before connecting and sending the HTTP request
+                      (Default is set by the KEDA_RESPONSE_HEADER_TIMEOUT environment
+                      variable)
```

Comment:
nit: looks like the `make manifests` didn't run successfully? Can you please try to re-run that, because this should read
```
How long to wait between when the HTTP request is sent to the backing app and when response headers need to arrive
```

Reply:
Good catch, @wozniakjan ! Fixed. I forgot to update the comment above the `ResponseHeader metav1.Duration`  in the `httpscaledobject_types.go` after a copy-paste and that's why the `make manifests` command produced the wrong description.